master/pkg/searcher/event_log.go

```diff
@@ -70,13 +72,18 @@ func (el *EventLog) TrialCreated(create Create, trialID int) {
 	el.uncommitted = append(el.uncommitted, trialCreated)
 	el.TrialIDs[create.RequestID] = trialID
 	el.RequestIDs[trialID] = create.RequestID
+	el.unitsCompletedByTrial[create.RequestID] = 0
```

Comment:
blocking: maybe`delete(el.unitsCompletedByTrial, create.RequestID)` in `TrialClosed` to save ourselves a bit?